### ISSUE TITLE
Where an API is both a GET and a POST there is an issue

### ISSUE THREAD

**Issue by jmons** [jmons/ramlwrap#21]
As part of porting tests to 'better' laid out RAML, I found this test case, but it doesn't work i.e. we have no users using this feature yet.

It seems that (in the use case:

```/web_app_tests:
    displayName: The Get Request (requires auth)
    get:
        description: Get a feature
        responses:
            200:
                description: A response
                body:
                    application/json:
                        example: '{ "status": "ok", "logged_in_user":"no_user"}'
    post:
        description: Post a feature
        responses:
            200:
                description: A response
                body:
                    application/json:
                        example: '{ "status": "ok", "logged_in_user":"no_user"}'        
```

in this instance, both GET and POST seem to be a `ValidatedPOSTAPI` rather then the right types (?)

**Comment by Jamian** [comment]
I'm currently working on a solution for this.

**Comment by jmons** [comment]
Fixed in @Jamian PR of #35
